bus-mapping/src/evm/opcodes/mload.rs

```diff
@@ -97,81 +97,71 @@ impl Opcode for Mload {
 mod mload_tests {
     use super::*;
     use crate::{
+        bytecode,
         evm::{
-            EvmWord, GasCost, GasInfo, Memory, OpcodeId, ProgramCounter, Stack,
+            EvmWord, GasCost, GasInfo, OpcodeId, ProgramCounter, Stack,
             StackAddress,
         },
-        BlockConstants, ExecutionTrace,
+        external_tracer, BlockConstants, ExecutionTrace,
     };
     use pasta_curves::pallas::Scalar;
 
+    macro_rules! gas_info {
+        ($gas:ident, $gas_cost: ident) => {{
+            #[allow(unused_assignments)]
+            GasInfo {
+                gas: {
+                    let temp = $gas;
+                    $gas -= GasCost::$gas_cost.as_usize() as u64;
+                    temp
+                },
+                gas_cost: GasCost::$gas_cost,
+            }
+        }};
+    }
```

Comment:
Moved this macro out, but left all other ones currently where they are (I don't want to risk increasing the diff too much in this PR because I may need to do quite a bit of rebasing in the near future :).